cpp/src/ray/test/cluster/test_cross_language_invocation.py

```diff
@@ -0,0 +1,21 @@
+import ray
```

Comment:
Could you add license declare for this file?

Reply:
The test file has no license in Java, keep the same with Java.